src/components/footnote-group/footnote-group.scss

```diff
@@ -0,0 +1,28 @@
+// TODO: Remove unused
```

Comment:
@Paul-Hebert Is this TODO still valid or can the comment be deleted?

Reply:
Oops! Good question. Lemme see...

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1518/commits/d958ab710bb08dc9366f079a41705ea703b81eee